JSDemos/Demos/ContextMenu/Templates/jQuery/styles.css

```diff
@@ -1,11 +1,22 @@
-.dx-menu-item-content span {
-  margin-right: 5px;
+.item-template-container {
+  display: flex;
+  align-items: center;
 }
 
-.dx-menu-item .dx-icon-spinright {
-  position: absolute;
-  top: 7px;
-  right: 2px;
+.dx-context-menu .dx-menu-item .dx-menu-item-content {
+  display: block;
+}
+
+.dx-menu-item-content span:not(.dx-icon-spinright) {
```

Comment:
Instead of adding margin style to specific element I suggest to use `gap` property to separate span elements inside `item-template-container` element
```
.item-template-container {
  display: flex;
  align-items: center;
  gap: 6px;
}
```